### DISCUSSION TITLE
Can't open cpeditor in macos

### DISCUSSION THREAD

**Discussion by joshirakshit30** [cpeditor/cpeditor#1224] [Q&A] [answered]
I'm a mac user. I downloaded it with the default version and asset given on the download page. After trying to open it, it shows an error of "macOS cannot verify that this app is free from malware." 
Any fixes??

**Comment by coder3101** [comment, marked as answer]
Its macOS security feature which blocks running apps that are not verified from Apple. [this article](https://www.howtoisolve.com/open-apps-from-an-unauthorized-developer-on-mac-os-x/) has step by step details on how to bypass it.

**Reply by joshirakshit30** [reply]
Thanks. That worked!
